src/mscorlib/shared/System/Span.NonGeneric.cs

```diff
@@ -688,6 +688,11 @@ public static ReadOnlySpan<char> AsReadOnlySpan(this string text, int start, int
             return new ReadOnlySpan<char>(ref Unsafe.Add(ref text.GetRawStringData(), start), length);
         }
 
+        // TODO: Delete once the AsReadOnlySpan -> AsSpan rename propages through the system
+        public static ReadOnlySpan<char> AsReadOnlySpan(this string text) => AsSpan(text);
```

Comment:
Do we need to temporarily add the AsReadOnlyMemory APIs too and update AsReadOnlyMemory -> AsMemory?

Reply:
Yes - as part of your other change, but it is not needed to get this PR go through.

Reply:
Ah I see. No one in this repo calls AsReadOnlyMemory.